tests/test_abstract_repr.py

```diff
@@ -260,7 +260,10 @@ def sequence(self, request):
         reg = Register(qubits)
         device = request.param
         seq = Sequence(reg, device)
-        seq.declare_channel("digital", "raman_local", initial_target="control")
+
+        seq.declare_channel(
+            "digital", "raman_local", initial_target=("control",)
```

Comment:
If you need use the `tuple` here why not below as well in the "rydberg_local"?

Reply:
I don't have to, but I can (the target can be a sequence of qubit IDs). I just made it a `tuple` to have both cases covered (ie when it's a single ID or a sequence of IDs - even though there is only one element here).

Reply:
Sounds good to me.